### PR TITLE
Merged

### PR DESCRIPTION
**Pull request by ResetReesi**
# Submitting Week 2 Assessment

### ✅ Completed Files

Select the following as confirmation that the files in this repository are complete.

- [✅ ] Coding Challenges
- [ ✅] Coding Concepts
- [ ✅] Interview Questions

### 📝 Feedback

If you would like specific feedback on a particular problem please leave details here:

### CODE REVIEW

File: code-challenges.test.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // ASSESSMENT 2: Coding Practical Questions with Jest
 
-const { workerData } = require("worker_threads")
-const { string } = require("yargs")
+// const { workerData } = require("worker_threads")
```
Comment (SunkissedQueen) [inline code-challenges.test.js:3]: caught the yargs!

**Review by SunkissedQueen — APPROVED** [review]
Brook, I will send my suggestions in slack. I cannot put comments next to things once the changes are merged.

**Comment by ResetReesi** [comment]
Sounds good

File: code-challenges.test.js (hunk 1 above)
Comment (ResetReesi) [reply code-challenges.test.js:3]: all I had to do was comment this out lol. I will try and use more higher order functions.

**Pull request merged**